packages/crayons-core/src/components/accordion-title/accordion-title.scss

```diff
@@ -38,7 +37,10 @@ $accordion-title-icon-color: var(--fw-accordion-title-icon-color, $color-seconda
 
   .accordion-title {
     flex-grow: 1;
-    line-height: var(--fw-accordion-title-line-height, 19px); // TODO: follow DSM
+    --fw-icon-color: var(
```

Comment:
the property should be `line-height` here. Can you remove the `--fw-icon-color` here ?

Reply:
Fixed.